src/client.cpp

```diff
@@ -1386,3 +1460,183 @@ int CClient::EstimatedOverallDelay ( const int iPingTimeMs )
 
     return MathUtils::round ( fTotalBufferDelayMs + iPingTimeMs );
 }
+
+// Management of Client Channels and mapping to/from Server Channels
+
+void CClient::ClearClientChannels()
+{
+    QMutexLocker locker ( &MutexChannels );
+
+    iActiveChannels = 0;
+    iJoinSequence   = 0;
+
+    for ( int i = 0; i < MAX_NUM_CHANNELS; i++ )
+    {
+        clientChannels[i].iServerChannelID = INVALID_INDEX;
+        // all other fields will be initialised on channel allocation
+
+        clientChannelIDs[i] = INVALID_INDEX;
+    }
+
+    // qInfo() << "> Client channel list cleared";
```

Comment:
Probably worth getting rid of this debug output.

Reply:
I thought it would be useful to retain the `qInfo()` messages I used while debugging, just commented out, then someone wanting to re-enable them only needs to uncomment, not compose afresh or search earlier commits for them.